BeagleBone/Black/pru/prugpio.h

```diff
@@ -0,0 +1,81 @@
+// This file defines the GPIO port addresses and PRU address
```

Comment:
Same include file in multiple places begs the question of why make it an include file.